optuna/pruners/successive_halving.py

```diff
@@ -158,3 +143,29 @@ def _completed_rung_key(rung):
     # type: (int) -> str
 
     return 'completed_rung_{}'.format(rung)
+
+
+def _get_competing_values(trials, value, rung_key):
+    # type: (List[FrozenTrial], float, str) -> List[float]
+
+    competing_values = [t.system_attrs[rung_key] for t in trials if rung_key in t.system_attrs]
+    competing_values.append(value)
+    competing_values.sort()
+    return competing_values
+
+
+def _is_value_promotable_to_next_rung(value, competing_values, reduction_factor, study_direction):
```

Comment:
This is a trivial comment, but I think the name of this function seems to conflict with the comment at L165.
The comment says `promote a trial` while the function name says `promote a value`. I'm not sure which one is more suitable, but I guess we can unify the expression.

Reply:
I aligned the words and slightly changed the comment. PTAL.